portfolio.html

```diff
@@ -0,0 +1,114 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="normalize.css" type="text/css" rel="stylesheet">
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<nav class="navigationBar fontStyleHead">
+			<a href="#topJump" class="fontStyleHead">Home</a>
+			<a href="#aboutMeJump" class="fontStyleHead">About</a>
+			<a href="#portfolioJump" class="fontStyleHead">Portfolio</a>
+			<a href="#footerJump" class="fontStyleHead">Socials</a>
+		</div>
+		</div>
+		
+	</nav>
+	
+	<section id="topJump" class="mainImage">
+		<div class="backgroundImage">
```

Comment:
fix your indentation, child elements should be indented inside parents

Reply:
Done